pkg/replica/backup.go

```diff
@@ -19,6 +20,7 @@ type ProgressState string
 const (
 	snapBlockSize = 2 << 20 // 2MiB
 
+	ProgressStateStarting   = ProgressState("starting")
```

Comment:
NIT: Use the const defined in `backupstore` instead.

Reply:
It seems you already added states into backupstore, so let's take this time to fix this. WDYT?

Reply:
Reuse the state in `backupstore`.
But some codes in `pkg/replica` are using `ProgressStateXXX`, so I still keep them and move to `pkg/replica/replica.go`.

Reply:
I feel some state constants are quite messy in the current code. NP, let's keep it for now.